dogs.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adoptadog</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet/styles.css">
+  </head>
+  <body>
+    <header>
+      <img src="adoptadog-logo.svg" alt="Adoptadog Logo">
+        <div class="checkout">
+            <p>$123.05</p>
+            <a href="checkout.html" class="checkout-button">Checkout</a>
+        </div>
+      <nav class="style-nav">
+        <ul>
+          <li class="links"><a href="index.html">Home</a></li>
+          <li class="links"><a href="dogs.html">Dogs</a></li>
+          <li class="links"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="row">
```

Comment:
These rows might not be the best method. It works fine for now, but what if you want to show 4 dog tiles wide on desktop, 3 on a tablet and 1 wide on a cell phone? Think of how you could leverage css grid or flexbox to lay these tiles out